### DISCUSSION TITLE
请问 Modal 组件在隐藏遮罩层后，可不可以在不关闭 Modal 的情况下，操作页面内非 Modal 元素

### DISCUSSION THREAD

**Discussion by JavanCheng** [ant-design/ant-design#46157] [Q&A]
场景：打开一个 Modal，填写 Modal 中的 Form，但是 Form 内容太多，需要从 Modal 外、页面内 copy 过来
需求：在不关闭 Modal 的情况下，复制 Modal 外页面内的文字

**Comment by zombieJ** [comment]
受控 open 不关闭就行了。

**Reply by Wxh16144** [reply]
这种感觉用抽屉 Drawer 更好，然后不设置 mask 再➕受控

**Reply by JavanCheng** [reply]
问题在于 modal 存在的时候无法操作 modal 外的元素，open 试了下不能解决这个问题呢

**Reply by linxianxi** [reply]
.ant-modal-wrap 加一个 point-events: none 就行了，Drawer 是有这个属性的
@zombieJ Modal 没有这个样式要看下是什么情况，会不会有问题
